src/marqo/s2_inference/s2_inference.py

```diff
@@ -182,39 +185,83 @@ def _update_available_models(model_cache_key: str, model_name: str, validated_mo
                                        f"Please wait for 10 seconds and send the request again.\n")
 
 
-def _validate_model_properties(model_name: str, model_properties: dict) -> dict:
-    """validate model_properties, if not given then return model_registry properties
+def validate_model_properties(model_name: str, model_properties: dict) -> dict:
+    """validate model_properties, if not given then return model_registry properties.
+
+    This is a rough validation as it only checks the minimum required fields and dimensions values. More indepth
+    check should be done when loading the model.
+
+    Raises:
+        InvalidModelPropertiesError: if the model_properties are invalid
+        UnknownModelError: if the model_name is not in the model registry
     """
     if model_properties is not None:
         """checks model dict to see if all required keys are present
         """
         required_keys = []
-        if model_properties.get("type", None) in (None, "sbert"):
+
+        if "type" not in model_properties:
+            base_error_message = ("You do not have a type key in your model_properties, so Marqo will try to load a "
+                                  "'sbert' model by default. If you want to load a different model, "
```

Comment:
This is raising an error but sounds like a warning?

Reply:
It will be returned as a part of the error message if the provided model properties can be loaded as an `sbert` model. Do you think this will the error message clearer and the user is not loading the `sbert` model without awareness?